package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "filmy-ts",
   "version": "0.1.0",
-  "private": true,
+  "private": false,
   "main": "src/index.tsx",
   "homepage": "https://awnigharbia.github.io/filmy/",
+  "engines": {
+    "node": ">0.12"
+  },
```

Comment:
This is too low. At least 12 should do the trick. `0.12 !== 12`.

Reply:
Ah didn't notice it